ClosedXML/Excel/Sparkline/XLSparkLineGroup.cs

```diff
@@ -0,0 +1,263 @@
+using System;
+using System.Collections.Generic;
+
+namespace ClosedXML.Excel
+{
+    internal class XLSparklineGroup : IXLSparklineGroup
+    {
+        public XLSparklineGroup(IXLWorksheet worksheet)
+        {
+            this.Worksheet = worksheet ?? throw new ArgumentNullException(nameof(worksheet));
+
+            AxisColor = XLColor.FromHtml("FF000000");
+            SeriesColor = XLColor.FromHtml("FF376092");
```

Comment:
I am surprised these are not theme colors but it seems valid.
I would only suggest using `XLColor.Black` instead of `XLColor.FromHtml("FF000000")`

Reply:
I will modify this so the default colors that are black are loaded from XLColor.black.